### PR TITLE
RSWEB-7046: multiple global nav improvements

### PR DESCRIPTION
**Pull request by runrog**
Things Done:
1.) Added down carots to desktop dropdown buttons, and right carots to mobile.
2.) Added toggle functionality for arrows on mobile when open/close 
3.) Added subtle bottom border to drop down links in mobile for differentiation purposes.
4.) Set a larger font size to all nav links in mobile.
I also switched our drop down trigger to 'clicks' instead of 'hovering' for a more consistent UX. This may still be up for debate. This also helps in developing for mobile/desktop as right now we're competing against the different listeners.

screen shot 2016-09-02 at 2 13 36 pm
screen shot 2016-09-02 at 2 13 56 pm
screen shot 2016-09-02 at 2 14 07 pm

### CODE REVIEW

File: styleguide/js/zoolander.js [hunk 1]
```diff
@@ -1,26 +1,55 @@
...
-  })
+
+    //reset dopdown carrets if collapsing the menu
+    if($(this).hasClass('collapsed')){
+      $topLevelDropDown.removeClass('navbar-dropDown-triggerActive');
+    }
+  });
+
+  //Toggle the open arrows on, top level, non drop down links
+  $('.navbar-topLink').not('.navbar-topLink.dropdown-toggle').click(function(){
```
Comment (SyntaxStacks) [inline styleguide/js/zoolander.js]: `$('.navbar-topLink:not(.navbar-topLink.dropdown-toggle)')`

**Comment by SyntaxStacks** [comment]
👍

File: styleguide/_themes/derek/scss/globalElements/mainnav.scss [hunk 2]
```diff
@@ -447,4 +469,18 @@
...
+      font-size: 1.45rem;
+
+      .navbar-dropDownLink,
+      .navbar-dropDownLabel,
+      .navbar-tertiary-dropDownLink {
+        border-bottom: 1px solid $gray-darker;
+        font-size: 1.45rem;
+      }
+    }
+  }
```
Comment (WidgetsBurritos) [inline styleguide/_themes/derek/scss/globalElements/mainnav.scss]: Can confirm the necessity of this three-level nesting?  Is this to override bootstrap specificity?

Comment (runrog) [inline styleguide/_themes/derek/scss/globalElements/mainnav.scss]: @WidgetsBurritos  Yup this grouping/nesting is already present to fight bootstrap specificity in the live nav sass file
screen shot 2016-09-06 at 3 17 27 pm

File: styleguide/derek/_partials/_mainnav.ejs [hunk 3]
```diff
@@ -36,7 +36,7 @@
                 <ul class="dropdown-menu navbar-dropDownMenu">
                   <li class="navbar-dropDownItem"><a href="javascript:void(0)" class="navbar-dropDownLink">Dedicated Hosting</a></li>
                   <li class="navbar-dropDownItem navbar-tertiary-dropDownTrigger">
-                    <span class="navbar-dropDownLink navbar-dropDown-trigger active">Compute</a>
+                    <span class="navbar-dropDownLink navbar-dropDown-trigger active">Compute</a></span>
```
Comment (WidgetsBurritos) [inline styleguide/derek/_partials/_mainnav.ejs]: You still have a bunch of unmatched closing `</a>` tags in here.

File: styleguide/js/zoolander.js [hunk 4]
```diff
@@ -44,7 +73,7 @@ jQuery(document).ready(function($) {
         .on('touchend', dropDownTrigger);
   } else {
     $(".navbar-tertiary-dropDownTrigger")
-      .hover(dropDownTrigger)
+      .on('click', dropDownTrigger)
       .on('click', function (e) { e.stopPropagation(); });
   }
```
Comment (WidgetsBurritos) [inline styleguide/js/zoolander.js]: If you're going to be changing JavaScript here, this is fine, but it's gonna a `www` PR as well:
https://github.rackspace.com/rswebteam/www/blob/master/styleguide/js/src/rackspace.js#L202

File: styleguide/_themes/derek/scss/globalElements/mainnav.scss [hunk 5]
```diff
@@ -365,6 +368,25 @@
...
+
+    &::after {
+      content: '\f0da';
+      font-family: FontAwesome;
+      margin-left: 10px;
+    }
+  }
+
+  .navbar-dropDown-triggerActive {
+    background-color: $gray-darker;
```
Comment (WidgetsBurritos) [inline styleguide/_themes/derek/scss/globalElements/mainnav.scss]: Can you set this background color on `:focus` and `:hover` as well? Bootstrap is setting this to transparent.

**Comment by WidgetsBurritos** [comment]
👍 on changes.

**Comment by WidgetsBurritos** [comment]
👍  on latest round of changes.

**Comment by ejchiefs** [comment]
👍

**Comment by WidgetsBurritos** [comment]
👍  for bug fix.

**Pull request merged**